courses/serializers.py

```diff
@@ -48,7 +48,25 @@ class Meta:
         fields = ["id", "title", "description", "thumbnail_url", "readable_id"]
 
 
-class CourseRunSerializer(serializers.ModelSerializer):
+class BaseCourseRunSerializer(serializers.ModelSerializer):
+    """Minimal CourseRun model serializer"""
+
+    class Meta:
+        model = models.CourseRun
+        fields = [
+            "title",
+            "start_date",
+            "end_date",
+            "enrollment_start",
+            "enrollment_end",
+            "expiration_date",
+            "courseware_url",
+            "courseware_id",
+            "id",
+        ]
+
+
+class CourseRunSerializer(BaseCourseSerializer):
```

Comment:
should this be `CourseRunSerializer(BaseCourseRunSerializer)`?

Reply:
😬